app/views/assets/_resource_main_content_right.html.erb

```diff
@@ -62,7 +62,7 @@
 <% end %>
 
 <% if resource.class.is_taggable? %>
-    <%= render :partial=>"assets/tags_box", :no_tags_message=>"Add tags (comma separated) ..." -%>
+    <%= render :partial=>"assets/tags_box", :locals => { :resource => resource }, :no_tags_message=>"Add tags (comma separated) ..." -%>
```

Comment:
update to use the more modern json like format, i.e

`partial: 'assets/tags_box', locals: { resource: resource } ...`